tests/integration/scale_column_test.py

```diff
@@ -19,7 +21,7 @@ def test_scale_column_pandas() -> None:
 
 
 @pytest.mark.skipif(
-    tuple(int(v) for v in pl.__version__.split(".")) < (0, 19, 0),
+    parse(pl.__version__) < Version("0.19.0"),
```

Comment:
This will help the tests work with release candidates, such as `polars==0.20.6rc1`